### PR TITLE
Doxysphinx docs

### PR DESCRIPTION
**Pull request by Maetveis**
Replaces: #3171

~~Requires https://github.com/RadeonOpenCompute/rocm-docs-core/pull/36~~ (merged)

### CODE REVIEW

**Comment by saadrahim** [comment]
@Maetveis the RTD build is not passing. Please help solve that.

**Comment by saadrahim** [comment]
I suspect the builds are failing as you separated out the doxygen fixes. Perhaps it was wrong advice to suggest that.

File: build-docs.sh
Comment (saadrahim) [inline build-docs.sh:1]: Should this be removed? I don't think it is necessary.

File: docs/.doxygen/Doxyfile [hunk 1]
```diff
@@ -32,19 +32,19 @@ DOXYFILE_ENCODING      = UTF-8
...
 
-PROJECT_NAME           = "HIP: Heterogenous-computing Interface for Portability"
+PROJECT_NAME           = "HIP Runtime API Reference"
 
 # The PROJECT_NUMBER tag can be used to enter a project or revision number. This
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         =
+PROJECT_NUMBER         = v3.0.1.0
```
Comment (saadrahim) [inline docs/.doxygen/Doxyfile]: Why is there a version number here?

File: build-docs.sh
Comment (Maetveis) [reply build-docs.sh:1]: Agreed, it was part of the original PR that I took over, removed.

File: docs/.doxygen/Doxyfile (hunk 1 above)
Comment (Maetveis) [reply docs/.doxygen/Doxyfile]: Removed now. I reviewed the changes to the doxyfile to only change relevant values.

**Review by chrispaquot — APPROVED** [review]

**Pull request merged**